### PR TITLE
Fix automatic paste not working on macOS

### LINKED ISSUE #2011: Pasting to Focussed window does not work on macOS
**Describe the bug**
Pasting to focussed window does not work.

**To Reproduce**
1. Start copyq in debug mode: `env COPYQ_LOG_LEVEL=DEBUG copyq`
2. Use global shortcut to activate copyq
3. Higlight first item and press enter

**Expected behavior**
The selected item should be pasted to the currently focussed window

**Actual Behaviour**
The selected item is moved to the clipboard, but is not automatically pasted to the focussed window. This happens with _all_ the apps I tested with so is not limited to a specific app.

**Version, OS and Environment**
```
CopyQ Clipboard Manager 6.1.0-50-g9317e2d0
Qt: 5.15.3
KNotifications: 5.87.0
Compiler: GCC
Arch: x86_64-little_endian-lp64
OS: macOS 10.15
```

**Additional context**
This is seen in the deubg logs:
```
DEBUG: Become background app
DEBUG: Raise a window "iTerm2"
DEBUG: Pasting item from main window to "iTerm2".
DEBUG: Searching key code for 'v'
DEBUG: KeyCode for 'v' is 9
DEBUG: Sending key codes 55 and 9
DEBUG: Saving items on application state change (2)
DEBUG: Can't convert between com.trolltech.anymime.public.utf16-plain-text and public.utf16-plain-text
DEBUG: Can't convert between public.text and public.utf16-plain-text
DEBUG: Can't convert between com.trolltech.anymime.text--plain and public.utf16-plain-text
DEBUG: Can't convert between dyn.agq80c6durvy0g2pyrf106p52fzv086d3se00875sqz3a and public.utf16-plain-text
DEBUG: Can't convert between com.trolltech.anymime.application--x-copyq-owner and public.utf16-plain-text
DEBUG: Can't convert between com.trolltech.anymime.application--x-copyq-owner and application/x-copyq-owner
DEBUG: Focus window is "iTerm2"
DEBUG: Executing: copyq onClipboardUnchanged
DEBUG: Executing: copyq onClipboardChanged
```

Note that this works fine for v4.1.0 but has never worked after that.
I have also tried building the app from source to no avail.

### PR DESCRIPTION
**Pull request by ajorpheus**
This reverts commit 02208bdef32432520d28badce3f7484d094c4725.

This fixes #2011, #1949

### CODE REVIEW

File: src/platform/mac/macplatformwindow.mm [hunk 1]
```diff
@@ -129,17 +129,19 @@ void sendShortcut(int modifier, int key, pid_t pid = -1) {
...
-            CGEventPost(kCGHIDEventTap, VUp);
-            CGEventPost(kCGHIDEventTap, commandUp);
+	    COPYQ_LOG( QStringLiteral("******* Not using CGEventPostToPid : %1").arg(pid));
+            /* CGEventPostToPid(pid, commandDown); */
+            /* CGEventPostToPid(pid, VDown); */
+            /* CGEventPostToPid(pid, VUp); */
+            /* CGEventPostToPid(pid, commandUp); */
         }
 
+        COPYQ_LOG( QStringLiteral("******* Using CGEventPost"));
```
Comment (hluk) [inline src/platform/mac/macplatformwindow.mm]: It's probably better to remove this completely (even with the `pid` argument), the logs nor the commented-out code look particularly useful.

**Review by hluk — CHANGES_REQUESTED** [review]
I don't use macOS so I cannot verify this, but it looks good if it works for you.

File: src/platform/mac/macplatformwindow.mm (hunk 1 above)
Comment (ajorpheus) [reply src/platform/mac/macplatformwindow.mm]: Done

**Comment by ajorpheus** [comment]
@hluk  resolved review comments, can you please have another look

**Pull request merged**